vendor/github.com/minio/minio/pkg/madmin/config-commands.go

```diff
@@ -129,3 +130,40 @@ func (adm *AdminClient) SetConfig(config io.Reader) (r SetConfigResult, err erro
 	err = json.Unmarshal(jsonBytes, &r)
 	return r, err
 }
+
```

Comment:
Shouldn't this change be made in minio/minio repository and vendored here?

Reply:
Yes. You're right.
I've added this change here for testing purposes. 
I already have the same change on the minio/minio side, but it is waiting for another PR minio/minio#6113 to be merged.